test/test.js

```diff
@@ -1178,6 +1178,17 @@ describe('Page', function() {
       expect(response.ok).toBe(true);
```

Comment:
Can we check the error text in this test? just to be safe

Reply:
Done.